### ISSUE TITLE
SSH Error

### ISSUE THREAD

**Issue by ReadieS** [0unknwn/auto-hotspot#6]
After running this script or using the instructions SSH appears to be broken upon reboot.  I unfortunatly do not have a micro hdmi cable yet but once I will experiment more. this is on a Pi 4B 2GB and 2021-05-07-raspios-buster-armhf

**Comment by 0unknwn** [comment]
Thanks for reporting it. Normally it shouldn’t break ssh. Was it connected to your local network or did a hotspot appear after rebooting it?

**Comment by ReadieS** [comment]
It was connected via WiFi the first time and via Ethernet the second time I attempted this. The second time I briefly saw the hotspot flash into my wifi list. Solar putty reported it was trying the connection and then stated "host hard reset". Same story both times after a restart and after trying to reconnect via the alternative method aka Ethernet or wifi which ever was opposite to the initial method.
As I had no other method to connect to the device and no cable to connect directly to the monitor I reinstalled Raspbian after each attempt.
